integration_tests/features/WalletRecovery.feature

```diff
@@ -1,15 +1,49 @@
 @wallet-recovery
 Feature: Wallet Recovery
 
-    @critical @broken
-    Scenario: Wallet recovery with connected base node staying online
-        Given I have a seed node NODE
-        And I have 1 base nodes connected to all seed nodes
-        And I have wallet WALLET_A connected to all seed nodes
-        And I have a merge mining proxy PROXY connected to NODE and WALLET_A with default config
-        When I merge mine 10 blocks via PROXY
-        When I mine 10 blocks on NODE
-        When I wait for wallet WALLET_A to have at least 55000000000 uT
-        Then all nodes are at height 20
-        When I recover wallet WALLET_A into wallet WALLET_B connected to all seed nodes
-        Then wallet WALLET_A and wallet WALLET_B have the same balance
+@critical @broken
+Scenario: Wallet recovery with connected base node staying online
+    Given I have a seed node NODE
+    And I have 1 base nodes connected to all seed nodes
+    And I have wallet WALLET_A connected to all seed nodes
+    And I have a merge mining proxy PROXY connected to NODE and WALLET_A with default config
+    When I merge mine 10 blocks via PROXY
+    When I mine 5 blocks on NODE
+    When I wait for wallet WALLET_A to have at least 55000000000 uT
+    Then all nodes are at height 15
+    When I recover wallet WALLET_A into wallet WALLET_B connected to all seed nodes
+    Then wallet WALLET_A and wallet WALLET_B have the same balance
+    And I have wallet WALLET_C connected to all seed nodes
+    And I send 100000 uT from wallet WALLET_B to wallet WALLET_C at fee 100
+    When I mine 5 blocks on NODE
```

Comment:
We need a wait of a second or so here otherwise mining starts before the transaction is finalized, especially on a faster system. This test failed on one system and passed on the other due to the confirmations only going to count of 2.

Reply:
hmm, perhaps I will rather put in a statement to wait for the transaction to be broadcast. I prefer that to adding in a finite wait which might not work on all setups.

Reply:
added `Then wallet WALLET_B detects all transactions are at least Broadcast`

Reply:
Yip, that did the trick.